src/mono/netcore/System.Private.CoreLib/src/System/Threading/TimerQueue.Browser.Mono.cs

```diff
@@ -53,14 +58,13 @@ private bool SetTimer(uint actualDuration)
             return true;
         }
 
-        private static void Run()
+        private static int PumpTimerQueue() // NOTE: this method is called via reflection by test code
         {
```

Comment:
This might need linker exclusions.

Reply:
it shouldn't since it's called by `TimeoutCallback()` which is kept with a `DynamicDependency` attribute on the `SetTimeout()` icall. I'm not sure what keeps `SetTimeout` alive though, but that was preexisting code.

Reply:
ah, I overlooked that it's called by `SetTimer()` which is called by the rest of the Timer code so that's why it's kept.